src/test/java/com/amazon/opendistroforelasticsearch/sql/esintgtest/AggregationIT.java

```diff
@@ -189,6 +189,30 @@ public void groupByTest() throws Exception {
         Assert.assertThat(gender.query(femaleBucketPrefix + "/COUNT(*)/value"), equalTo(493));
     }
 
+    @Test
+    public void groupByHavingTest() throws Exception {
```

Comment:
Could you also verify if JDBC or CSV formatter works?

Reply:
Done. Add the IT aggregationFunctionInHaving in PrettyFormatResponseIT.